iOS/Controllers/CoreDataTableViewDataSource.swift

```diff
@@ -79,6 +79,8 @@ class CoreDataTableViewDataSource<Delegate: CoreDataTableViewDataSourceDelegate>
             break
         case .update:
             break
+        @unknown default:
+            fatalError()
```

Comment:
Please don't let the app crash, it would be better to use `break` here.
The same goes for the following changes.